src/quo2/foundations/colors.cljs

```diff
@@ -60,6 +60,7 @@
 
 ;;Blur
 (def neutral-5-opa-70 (alpha neutral-5 0.7))
+(def neutral-80-blur-opa-80 "rgba(25,36,56,0.8)")
```

Comment:
Hi @ulisesmac is there a reason why the name of this color is `neutral-80-blur-opa-80` and not `neutral-80-opa-80` and you are not using the `alpha` function like in the other colors?